docs/environment.yml

```diff
@@ -5,15 +5,19 @@ dependencies:
   - ansi2html
   - autodoc-traits
   - bqplot >=0.12.20
+  - comm <0.1.3
   - docutils >=0.19
   - doit-with-toml
   - flit >=3.7.1,<4
   - git
-  - ipywidgets >=8.0.4,<9
+  - ipykernel <6.22.0
+  - ipywidgets <8.0.5
```

Comment:
are these two `ipywidgets` lines intentional?

Reply:
Yep, bottom pin is set in `_run.yml` and `lab3.5.yml` sets a top pin. not worried about `9` right now, let's deal with the breaking changes in `8.x +/ 1`